src/Sylius/Behat/Context/Setup/CatalogPromotionContext.php

```diff
@@ -81,6 +84,21 @@ public function itWillBeAppliedOnTaxon(CatalogPromotionInterface $catalogPromoti
         // TODO: Add Taxon Rule to the Catalog Promotion
     }
 
+    /**
+     * @Given /^(it) will be applied on ("[^"]+" variant)$/
+     */
+    public function itWillBeAppliedOnVariant(CatalogPromotionInterface $catalogPromotion, ProductVariantInterface $variant): void
+    {
+        $catalogPromotionRule = new CatalogPromotionRule();
+
+        $catalogPromotionRule->setType(CatalogPromotionRule::TYPE_CONTAINS_VARIANTS);
```

Comment:
Maybe it's also a good idea to introduce per-type create methods? Like `createContainsVariant(array $configuration)`? But it can be an improvement